### PR TITLE
Author profile page

### PR DESCRIPTION
**Pull request by anishjangra-dev**
- Author profile page

### CODE REVIEW

File: src/components/AuthorProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    </Container>
+  );
+};
+
+const AuthorProfile = () => {
+  const { id } = useParams();
+
+  const [authorProfileData, setAuthorProfileData] = React.useState<Author>();
+  const [authorProfileLoading, setAuthorProfileLoading] = React.useState(false);
+  const [authorProfilePresent, setAuthorProfilePresent] = React.useState(true);
```
Comment (SourabhJaz) [inline src/components/AuthorProfile.tsx:35]: Fetch sessions by the author also.
`curl --location --request GET 'https://sourabhjaz.pythonanywhere.com//api/session?author=3'`

File: src/components/AuthorProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            >
+              {authorProfileData.image_url ? undefined : Utils.getAuthorInitials(authorProfileData.name)}
+            </Avatar>
+            <Box>
+              <Typography variant="h3" marginBottom="5px">
+                {authorProfileData.name}
+              </Typography>
+              <Typography variant="body1" color="text.secondary" gutterBottom fontStyle="italic">
+                {`${authorProfileData.outline}`}
+              </Typography>
```
Comment (SourabhJaz) [inline src/components/AuthorProfile.tsx:91]: We need to add a <LatestSessions /> section here

File: src/components/AuthorProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+              src={authorProfileData.image_url || '/static/images/avatar/default.jpg'}
+              alt={authorProfileData.name}
+              sx={{
+                width: 84,
+                height: 84,
+                marginRight: 2,
+              }}
+            >
+              {authorProfileData.image_url ? undefined : Utils.getAuthorInitials(authorProfileData.name)}
+            </Avatar>
```
Comment (SourabhJaz) [inline src/components/AuthorProfile.tsx:84]: Linked in URL should also be shared here.

Comment (anishjangra-dev) [reply src/components/AuthorProfile.tsx:84]: done

File: src/components/AuthorProfile.tsx (hunk 1 above)
Comment (anishjangra-dev) [reply src/components/AuthorProfile.tsx:35]: done

File: src/components/AuthorProfile.tsx (hunk 2 above)
Comment (anishjangra-dev) [reply src/components/AuthorProfile.tsx:91]: what info will it contain ?

**Review by SourabhJaz — APPROVED** [review]

**Pull request merged**